__tests__/components/AppLayout/SideBar.test.tsx

```
@@ -215,6 +271,14 @@ const mockNavItems: NavBarItem[] = [
 describe("SideBarMenu", () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+    mockLogout.mockReset();
+  });
+
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
   beforeEach(() => {
     mockPush.mockClear();
   });
```

Biome (2.1.2)

[error] Disallow duplicate setup and teardown hooks.

Disallow beforeEach duplicacy inside the describe function.

(lint/suspicious/noDuplicateTestHooks)